src/Model/ValueObject/Price/Tariffs.php

```diff
@@ -5,11 +5,39 @@
 namespace CultuurNet\UDB3\Model\ValueObject\Price;
 
 use CultuurNet\UDB3\Model\ValueObject\Collection\Collection;
+use CultuurNet\UDB3\Model\ValueObject\Translation\Language;
 
 class Tariffs extends Collection
 {
     public function __construct(Tariff ...$tariffs)
     {
         parent::__construct(...$tariffs);
     }
+
+    public function hasDuplicates(): bool
```

Comment:
```suggestion
    public function hasDuplicateNames(): bool
```
Small nit but this might be clearer, as a duplicate can also be interpreted as same name + same price + same currency.

Reply:
Method is no longer used.